installer.sh

```diff
@@ -104,8 +107,8 @@ cd $DIR
 # glog
 git clone https://github.com/google/glog.git
 cd glog
-mkdir build
-cd build
+mkdir tmp
+cd tmp
```

Comment:
what's wrong with build

Reply:
On my mac it complains that build already exists

Reply:
instead of creating /tmp, maybe we can just cd in to build without making it.

Reply:
heh so apple's HFS is case-insensitive by default, and that's why it's conflicting with the file `BUILD` in `glog/`

what do

Reply:
We could just do `mkdir -p build`.
Adding the -p should stop it from complaining if the `build/` directory already exists

Reply:
I still get `mkdir: build: Not a directory`